CMakeLists.txt

```diff
@@ -252,6 +252,7 @@ target_link_libraries(
   kvsWebrtcClient
   PRIVATE kvspicUtils
           kvspicState
+          m
```

Comment:
Can you describe why we need libm in case of the webrtc library? I wasn't sure whether this is needed